src/Equinor.ProCoSys.IPO.Domain/AggregateModels/InvitationAggregate/Invitation.cs

```diff
@@ -197,7 +197,7 @@ public void CompleteIpo(Participant participant, string participantRowVersion, P
             CompletedBy = completedBy.Id;
             CompletedAtUtc = completedAtUtc;
             AddPreSaveDomainEvent(new IpoCompletedEvent(Plant, ObjectGuid));
-            AddPostSaveDomainEvent(new Events.PostSave.IpoCompletedEvent(Plant, ObjectGuid));
+            AddPostSaveDomainEvent(new Events.PostSave.IpoCompletedEvent(Plant, ObjectGuid, this));
```

Comment:
Invititationtests has one test named CompleteIpo_ShouldAddCompleteIpoEvent
This should be 2 tests:
CompleteIpo_ShouldAddCompleteIpoPreSaveEvent
CompleteIpo_ShouldAddCompleteIpoPostSaveEvent

@llykkelarsen Add a TECH to verify if these tests are forgotten elsewhere (or @janhagevold fix it now all places)